src/dialogs/SimpleTwoActionsDialog/SimpleTwoActionsDialog.js

```diff
@@ -69,6 +98,26 @@ SimpleTwoActionsDialog.propTypes = {
    *  ## Function used when button2 is clicked
    */
   handleClickOnButton2: PropTypes.func.isRequired,
+
+  /**
+   * Props to add/override of dialog component
```

Comment:
maybe a missing word: _add/override SOMETHING of dialog component_ ? :slightly_smiling_face: